src/windows/process.rs

```diff
@@ -221,8 +222,18 @@ impl CPUsageCalculationValues {
             old_process_user_cpu: 0,
             old_system_sys_cpu: 0,
             old_system_user_cpu: 0,
+            nb_cpus: 0,
         }
     }
+
+    fn total_accumulated_cpu_usage(&self) -> f32 {
+        self.old_process_user_cpu
+            .saturating_add(self.old_process_sys_cpu) as f32
+            / self
+                .old_system_user_cpu
+                .saturating_add(self.old_system_sys_cpu) as f32
+            * self.nb_cpus as f32
+    }
```

Comment:
Again, I don't think this is right. It should only return `old_process_user_cpu + old_process_sys_cpu`. The other parts are dividing with the global CPU time and number of CPUs, which just isn't right here.